frontend/packages/ceph-storage-plugin/src/utils/dashboard.ts

```diff
@@ -1,11 +1,11 @@
 import * as _ from 'lodash';
 import { PrometheusResponse } from '@console/internal/components/graphs';
 
-export const getResiliencyProgress = (results: PrometheusResponse): number => {
+export const getResiliencyProgress = (results: PrometheusResponse): string => {
   const progress = _.get(results, 'data.result[0].value[1]');
```

Comment:
1. Define type for progress.

```suggestion
  const progress: string = _.get(results, 'data.result[0].value[1]', null);
```

2.
```
if(progress) {
  return (Number(convertedProgress) * 100).toFixed(1);
}
return null;
```

Reply:
1. We are trying to avoid now returning `null` as a value from promethues responses, because it already return `undefined`. ( have removed it too ) 

2. Prometheus returns string as a response , so `NaN` will be `'NaN'` , hence not relying on `truthy/falsy` values.
(In this case 'NaN' can come)

I can add type indeed here and the pther place you mentioned.
Thanks!